include/library/spdm_lib_config.h

```diff
@@ -7,6 +7,121 @@
 #ifndef SPDM_LIB_CONFIG_H
 #define SPDM_LIB_CONFIG_H
 
+/* Code space optimization for optional messages.
```

Comment:
why move the location?

Reply:
For `#if LIBSPDM_ENABLE_CAPABILITY_PSK_CAP` in line 151.